react/details/src/i18n/i18n.js

```diff
@@ -3,9 +3,15 @@ import { initReactI18next } from 'react-i18next';
 
 import locales from 'i18n/locales';
 
+// adds the default namespace ('translation') to resources object per i18next documentation
```

Comment:
can we use the same i18n code already used for other widgets? it should help with the generation as well

Reply:
I have this covered in my generation code already. can I leave this be and just update from the generated code later?

Reply:
sure